### PR TITLE
Update TetheredPlacement.cs to use local pose

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

As the scene can be dragged around, the platform the objects should sit on can move around, causing the objects to respawn into nothingness.

This change allows the objects to remember their relative poses and correctly spawn back above the platform.

### CODE REVIEW

File: Assets/MRTK/Examples/Demos/HandTracking/Scripts/TetheredPlacement.cs [hunk 1]
```diff
@@ -11,26 +12,35 @@ namespace Microsoft.MixedReality.Toolkit.Examples.Demos
...
+            distanceThresholdSquared = distanceThreshold * distanceThreshold;
         }
 
         private void LateUpdate()
         {
-            float distanceSqr = (RespawnPoint - this.transform.position).sqrMagnitude;
+            float distanceSqr = (localRespawnPosition - transform.localPosition).sqrMagnitude;
 
-            if (distanceSqr > DistanceThreshold * DistanceThreshold)
+            if (distanceSqr > distanceThresholdSquared)
```
Comment (davidkline-ms) [inline Assets/MRTK/Examples/Demos/HandTracking/Scripts/TetheredPlacement.cs:43]: With this change, updates to DistanceThreshold do not get applied (distanceThresholdSquared is only calculated in Start)

**Review by davidkline-ms — CHANGES_REQUESTED** [review]

File: Assets/MRTK/Examples/Demos/HandTracking/Scripts/TetheredPlacement.cs (hunk 1 above)
Comment (keveleigh) [reply Assets/MRTK/Examples/Demos/HandTracking/Scripts/TetheredPlacement.cs:48]: Updated!

**Review by davidkline-ms — APPROVED** [review]

**Pull request merged**